src/api/chronicler.js

```diff
@@ -30,4 +30,46 @@ export const initChron = () => {
         feedMe(scheduleToFeed([r.data]));
       }
     });
+  getNames()
+    .then(res => {
+      addFilterOptions(Object.entries(res).map((n) => ({value: {filters: n }, label: n[1]})))
+    });
+
+  fetch(`https://api.sibr.dev/chronicler/v1/teams`)
+    .then(res => res.json())
+    .then(res => {
+      addFilterOptions(res.data.filter(d => (
+        ilbTeamIds.indexOf(d.id) !== -1
+      )).map((d) => (
+        {value: {filters: [d.id, d.data.nickname, d.data.fullName] }, label: d.data.fullName}
+      )));
+    })
 };
+
+// stolen from https://github.com/xSke/blaseball-player-list/blob/main/src/teams.ts thanks astrid
+const ilbTeamIds = [
```

Comment:
this works but also we could use this API if we wanted to futz around with yet another thing https://api.blaseball-reference.com/docs#/Teams/get_v2_teams

Reply:
That is much better than this